src/service/action-impl.js

```diff
@@ -61,6 +61,35 @@ const ELEMENTS_ACTIONS_MAP_ = {
   'form': ['submit'],
 };
 
+/**
+ * Interactable widgets which should trigger tap events when the user clicks
+ * or activates via the keyboard. Not all are here, e.g. progressbar, tabpanel,
+ * since they are readonly or composite widgets that shouldn't need to trigger
+ * tap events on their own.
+ * See https://www.w3.org/TR/wai-aria-1.1/#widget_roles
```

Comment:
How did you select which roles to add below? E.g. `combobox` is listed under "composite" in this link.

Reply:
My criteria were similarity to existing components that should receive `tap`. Comboboxes typically behave like HTML `select` but with a text typeahead, so I imagined it would fit with this issue fixing `amp-selector`. Thinking about that particular widget now, we might not want space to trigger tap in an element with a text input. Were there any others present that you think should be omitted, or any that you think should be added?

Reply:
No, just curious.